src/nuxeo-cloud/nuxeo-cloud-customer-console.md

```diff
@@ -0,0 +1,56 @@
+    - nuxeo-cloud
+tree_item_index: 350
+---
+
+## Nuxeo Cloud Console
+The Nuxeo Cloud Customer Console provides our customers with increased self service capabilities. Among other useful features, the Cloud Customer Console primarily gives users the ability to build Dedicated Customer Images and deploy them to Customer Pre-Production Environments. Our long term goal is to enable more self service capabilities allowing customer increased visibility to hosted Nuxeo Cloud environments and related information.
+### Dedicated Image Build & Deployment Pipelines
+The Nuxeo Cloud Customer Console introduces the ability to build and deploy a dedicated Nuxeo Docker Image to your Pre-Production Cloud Environment.
+-	**Image Build Pipeline** - The Image Build Pipeline starts with an LTS 2021 + HotFixes Docker image. Then combines it with your application Marketplace package and dependent add-ons to produce a Dedicated Customer Image. This Image Build is defined and managed as part of a Customer Project profile. This Dedicated Docker Image artifact can then be deployed to your Pre-Production & Production environments.
+-	**Image Deployment Pipeline** - The Deployment pipeline is the action of deploying a Nuxeo Docker Image created by the Build Pipeline to a Pre-Production or Production environment. The same Image Build and Deployment process is used by the Nuxeo Cloud team to deploy Customer specific Nuxeo Docker Images artifacts to customer environments. The Nuxeo Docker Image is the artifact that is deployed and each image has a unique build number. The Nuxeo Cloud Customer Console references the build number to select and deploy to the Environment.
+-	**Build Number**
+All customers should provide the Build Number as part of all Production deployment Jira requests going forward. The Build Number refers to the Nuxeo Docker Image you have tested in your Pre-Production environment and want deployed to Production.
+### Overview Page
+After logging into the Cloud Console, users will land on the Overview page which provides the following features:
+1.	Add, Remove, Edit and View Customer Contact Information
+2.	Link to Nuxeo Cloud documentation
+3.	List of Production and Pre-Production Environments
+ 
+Environment details can be accessed by clicking on the “eye” icon shown next to available environments listed on the Overview Page. At this time customers can only view and schedule deployments to Pre-Production environments within the Cloud Console. Production deployment requests must be made by submitting a Jira SUPNXP ticket. 
+#### Product Version & Product Revision
+For each environment, we display the Product Version and the Nuxeo Product Revision. These provide information on the latest HotFix deployed to each environment. 
+Please note that versions are dynamically retrieved from the login page. If a customer uses a custom login page instead of the default Nuxeo login page, we will not be able to retrieve the version and will display Not Available.
```

Comment:
```suggestion
Please note that versions are dynamically retrieved from the login page. If a customer uses a custom login page instead of the default Nuxeo login page, we will not be able to retrieve the version and will display _Not Available_.
```